packages/preview-middleware-client/src/cpe/outline/nodes.ts

```diff
@@ -136,15 +142,40 @@ export async function transformNodes(input: OutlineViewNode[], scenario: Scenari
     return items;
 }
 
+/**
+ * Fill reuse components ids.
+ *
+ * @param reuseComponentsIds ids of reuse components that are filled when outline nodes are transformed
+ * @param node view node
+ * @param scenario type of project
+ */
+async function fillReuseComponents(
+    reuseComponentsIds: Set<string>,
+    node: OutlineViewNode,
+    scenario: Scenario
+): Promise<void> {
+    const version = sap.ui.version;
```

Comment:
Global usage should be avoided. Please use `sap/ui/VersionInfo` instead.

Reply:
fixed in #[9aa908c](https://github.com/SAP/open-ux-tools/pull/1941/commits/9aa908cf6e0415b18d849970bd855e53f98ef3db)